src/titiler/core/titiler/core/middleware.py

```diff
@@ -95,30 +93,13 @@ async def send_wrapper(message: Message):
         await self.app(scope, receive, send_wrapper)
 
 
-AVAILABLE_LOG_DATA = Literal[
-    "method", "referer", "origin", "path", "path_params", "query_params", "headers"
-]
-
-
-def create_logger() -> logging.Logger:
-    """Create titiler logger."""
-    logger = logging.getLogger("titiler")
-    logger.setLevel(logging.INFO)
-    handler = logging.StreamHandler(sys.stdout)
-    handler.setLevel(logging.INFO)
-    handler.setFormatter(logging.Formatter("%(name)s - %(message)s"))
-    logger.addHandler(handler)
-    return logger
-
-
 @dataclass(frozen=True)
 class LoggerMiddleware:
     """MiddleWare to add logging."""
 
     app: ASGIApp
-    logger: logging.Logger = field(default_factory=create_logger)
-    log_data: List[AVAILABLE_LOG_DATA] = field(
-        default_factory=lambda: list(get_args(AVAILABLE_LOG_DATA))
+    logger: logging.Logger = field(
+        default_factory=lambda: logging.getLogger("titiler-requests")
```

Comment:
I choose to set the `name` directly to be a be more precise than `titiler.core.middleware`

Reply:
I think it would be wise to at least put it behind `titiler` (ie `titiler.requests`) that way a person could apply a blanket logging policy to all of `titiler` or one purely targeting the requests logger. As currently written, implementors wouldn't be able to target titiler & the titlier-requests with a single statement.

Reply:
I'm not sure to get what you're saying. would you prefer `titiler.requests` ?

Reply:
yes, the idea being that if they shared the same root logger (ie `titiler`), someone could customize the logger for `titiler` (which would include all of the `titiler.requests` logs) or they could do separate loggers for `titiler` and `titiler.requests` if they wanted that (setting `propogate=False` to keep the `titiler.requests` logs from going to the `titiler` log handler).